internal/controller/server_controller.go

```diff
@@ -436,6 +436,7 @@ func (r *ServerReconciler) updateServerStatus(ctx context.Context, log logr.Logg
 	server.Status.Manufacturer = systemInfo.Manufacturer
 	server.Status.Model = systemInfo.Model
 	server.Status.IndicatorLED = metalv1alpha1.IndicatorLED(systemInfo.IndicatorLED)
+	server.Status.TotalSystemMemory = &systemInfo.TotalSystemMemory
```

Comment:
We should use `ptr.To()` when assigning pointer values to fields.

Reply:
Why? `ptr.To()` is literally `return &val`. `ptr.To` is useful to get a pointer to a literal in an argument for a method call, for instance. Here it's just more keypresses. 🤔